node-runner-cli/radixnode.py

```diff
@@ -24,7 +24,7 @@
 from setup import Base, Docker, SystemD
 from utils.utils import Helpers
 from utils.utils import run_shell_command
-from version import __version__
+#from version import __version__
```

Comment:
This commented line should be removed.

Out of interest, why was this removed? Or was it giving us the python version, not the package verison? :)

Reply:
I removed it by accident...